### DISCUSSION TITLE
Paragraph 的 copyable 可以支持异步吗？

### DISCUSSION THREAD

**Discussion by crazyair** [ant-design/ant-design#47860] [General]
是这样的，页面有个按钮，点击按钮后才会发起请求，请求返回的内容，复制到剪切板

```tsx
    <Paragraph copyable={{ text: 'Hello, Ant Design!' }}>Replace copy text.</Paragraph>
```
看到都是这样使用，无法满足需求，只能再单独安装 copy 库来实现

**Comment by crazyair** [comment]
这个本来看到的 `Replace copy text.` 跟 copy 的 `Hello, Ant Design!` 内容不同，这个也好支持

`<Paragraph copyable={{ text: async () = > await xxx }}>Replace copy text.</Paragraph>` 

@zombieJ

**Comment by crazyair** [comment]
https://github.com/ant-design/ant-design/pull/48123

**Comment by zombieJ** [comment]
会上聊是感觉这个过于业务属性，需要补充一下是什么场景复制内容和文本不一致。

**Reply by crazyair** [reply]
这个的用例就是显示跟复制的文本不一致的场景呀，只是不是异步方式 
![image](https://github.com/ant-design/ant-design/assets/7971419/82e96614-61f0-4245-a22d-7b140a3446a1)

**Reply by zombieJ** [reply]
就是为什么要异步不一致

**Reply by crazyair** [reply]
在表格内使用，分页接口没有返回复制的内容，需要用户点击复制的时候请求复制的内容

**Reply by zombieJ** [reply]
能更具体一些不？文案是什么，复制出来的是什么？

**Reply by crazyair** [reply]
```tsx
<Form.Item name='id'><Select /></Form.Item>
<Text copyable />
```
要根据 id 动态选择，需要用 id 调接口返回详情的一个 context 字段，复制这个字段的内容

**Reply by crazyair** [reply]
还有显示的手机号的密文，点击复制要复制明文，明文默认不返回，复制的时候要调接口返回

**Reply by zombieJ** [reply]
联动是吧，那挺合理的
